src/components/WebexLocalMedia/WebexLocalMedia.jsx

```diff
@@ -14,16 +14,20 @@ import {
 } from '../hooks';
 
 /**
- * Webex Local Media component displays the user's local video.
+ * Webex Local Media component displays the user's local video or local share.
+ * If the meeting is active and the current user is sharing the screen,
+ * the local share is displayed; otherwise, the local video is displayed (if video is enabled).
  *
  * @param {string} props.meetingID  ID of the meeting from which to obtain local media
  * @returns {Object} JSX of the component
  */
 export default function WebexLocalMedia({className, meetingID}) {
   const [mediaRef, {width}] = useElementDimensions();
-  const {localVideo} = useMeeting(meetingID);
+  const {localVideo, localShare, remoteVideo} = useMeeting(meetingID);
+  const isActive = remoteVideo !== null;
   const {ID} = useMe();
-  const videoRef = useStream(localVideo);
+  const localStream = (isActive && localShare) || localVideo;
```

Comment:
I think a more scalable solution for this is to move the logic of deciding what type of local media to show to [In-meeting component](https://github.com/webex/components/blob/master/src/components/WebexInMeeting/WebexInMeeting.jsx), as this component decides what the experience should be when you're in a meeting.
I'm thinking that maybe we can add a prop to local media `mediaType` where an user can pass `video`, `audio` or `screen` and that's how we can determine the type of local media to show

Reply:
Initially, we wanted to have two separate components `LocalVideo` and `LocalShare` so that `InMeeting` could choose one of the 2. However, we then saw that `RemoteMedia` displays both share and video, and has some logic of which to display where. So we updated this `LocalMedia` component in a similar way. I still think that, for now, it would be ok to let this component take care of the logic and maybe move it up when the need arises.

Reply:
You're right in that `WebexRemoteMedia` has some meeting logic in it, that looking back, shouldn't be there, it should be in `WebexInMeeting` instead. I don't want to make the same mistake with local media, as I foresee some near-team use cases where users just want to get a hold of media display only. Putting the logic there now would basically diminish the usefulness of having it as a component.